examples/integrations/cdi/datasource-hikaricp/README.adoc

```diff
@@ -0,0 +1,108 @@
+= Hikari Connection Pool Integration Example
+
+== Overview
+
+This example shows a trivial Helidon MicroProfile application that
+uses the Hikari connection pool CDI integration, shows how to package
+it into a Docker image, and shows how to run it in a Docker container.
+It also shows how to run the Oracle database in a Docker container and
+connect to it using the application's container.
+
+== Notes
+
+To create a user-defined Docker network:
+
+```sh
+$ docker network create helidon-examples
+```
+
+To log in to the Oracle Container Registry (which you will need to do
+in order to pull Oracle database images from it):
+
+```sh
+$ docker login -u your.name@yourdomain.com -p yourpasswordhere container-registry.oracle.com
+```
+
+To run Oracle's `database/standard` Docker image in a Docker container
+named `mysql` connected to this network that publishes its ports to
```

Comment:
Should be oracle instead of mysql.

Reply:
Good catch; I haven't looked at the Oracle example yet but looks like I should!